File: Week 1/hacker/mario.c
#include <stdio.h>

int main(void)
{
    // get height and validate
    int height;
    printf("Height: ");
    scanf("%i", &height);

    while(height < 0 || height > 23)
    {
        printf("Retry:");
        scanf("%i", &height);
    }

    // print
    for (int spaces = height - 1; spaces >= 0; spaces--)
    {
        for (int i = 0; i < spaces; i++)
            printf(" ");

        for (int j = 0; j < height - spaces; j++)
            printf("#");

        printf(" ");

        for (int j = 0; j < height - spaces; j++)
            printf("#");

        printf("\n");
    }
}

File: Week 8/mario.py
# get height
height = int(input("height: "))
while height not in range(1, 24):
    height = int(input("Retry: "))

# print pyramid
for i in range(height, 1, -1):
    # print spaces
    print(" " * (i - 1), end="")
    # prints hashes
    print("#" * (height - (i - 2)))
